fast-list.js

```diff
@@ -72,42 +63,108 @@ function FastList(source) {
   on(this.els.container, 'click', this);
   on(window, 'resize', this);
 
-  this.rendered = schedule.mutation(function() {
-    this.updateContainerGeometry();
-    this.updateListHeight();
+  // Create a list element if one wasn't provided
+  if (!this.els.list) this.els.list = document.createElement('ul');
+  if (!this.els.itemContainer) this.els.itemContainer = this.els.list;
+
+  this.rendered = schedule
+    .mutation(this.renderPhase1.bind(this))
+    .then(this.renderPhase2.bind(this))
+    .then(function() {
+
```

Comment:
I don't think it would be possible to scroll in this microtask time period would it?